src/pipeline/pipeline.hpp

```diff
@@ -32,18 +32,13 @@ class Pipeline : public Producer<Product>, public Consumer<Consumable> {
     this->result = None();
 
     // Start pipeline.
+    // TODO(skonefal): Pipeline should not "return" with error.
     Try<Nothing> ret = this->produce(_product);
 
-    Try<Nothing> retPostRun = this->postPipelineRun();
```

Comment:
So what's the current status of error handling? Do we always go through all filters to the end of the pipeline?